package.json

```diff
@@ -18,12 +18,13 @@
     "start": "gulp --dev",
     "webpack": "webpack --config webpack.static.config.js --mode development && webpack --config webpack.build.config.js --mode development",
     "build": "npm run prestart && npm run webpack",
+    "build-docs": "npm run prestart && gulp build",
     "lint-css": "./node_modules/.bin/stylelint \"src/assets/sass/**/*.scss\" ",
     "lint-js": "./node_modules/.bin/eslint \"src/assets/js/**/*.js\" \"gulpfile.js/**/*.js\" \"webpack.static.config.js\" \"webpack.build.config.js\" ",
     "lint": "npm run lint-js && npm run lint-css ",
     "test": "./node_modules/.bin/karma start ./tests/karma.conf.js",
     "pretest": "npm run build",
-    "docs": "doctoc --title \"### Contents\" --maxlevel 2 docs/README.md",
+    "readme": "doctoc --title \"### Contents\" --maxlevel 2 docs/README.md",
```

Comment:
Renamed this command as it's associated with the github readme, rather than the docs site